content/integrations/agents/LINUX-agent/LINUX-standard-install.md

```diff
@@ -8,9 +8,31 @@ author: Lawrence Lane
 #pre: "<i class='fa fa-download'></i> &nbsp;"
 weight: 1
 ---
-This integration’s package (computed metrics, dashboards, and policies that will give you important events and alerts) will be automatically enabled and provisioned to your account as soon as CloudWisdom receives data from the integration. The PACKAGES button on the integration setup page will become active once data is received, so you’ll be able to disable and re-enable the package at will.
+This integration’s package (computed metrics, dashboards, and policies) is automatically enabled and provisioned to your account as soon as CloudWisdom receives data.
 
-## 1. Copy Install Command From Linux Integration Setup Page
+## Cost Optimization Quick Setup
+
+Customers using **only Cost Optimization features**, such as Bill Analysis, can install the Linux Agent with just one quick command! This method enables the Linux Agent's [Simple Collector][2].
+
+{{% notice tip %}}
+
+Remember to update your Linux Agent config file if you start using CloudWisdom's Capacity Monitoring tools.
+
+{{% /notice %}}
+
+1. Navigate to your Linux Agent integration in Metricly.
+2. Copy the **API Key**.
```

Comment:
The API key will be pre-generated for them in the one-line command so they need not copy it.